### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by brandscill** [CoreForms/docs#1] [Q&A]
Hi,

I have a very small webforms project that I want to migrate as a trial. I can't find anything in the wiki with regards to getting started.

Do I first want to use the .NET upgrade tool to go to .NET 7?

What are the steps here?

**Comment by MichaelKetting** [comment]
Hi @brandscill, 

the commit history of the https://github.com/CoreForms/WingtipToys-Demo is set up as a step by step guide for the web application. As for the rest of the solution: yes, the .NET upgrade tool and migration guide lines by Microsoft would be a great way to get started.

**Reply by brandscill** [reply]
Okay thanks. I'll get started later and see where we end up

**Reply by brandscill** [reply]
The .NETFramework Bridge, how much milleage has that had with third party libraries? We use AjaxPro in our current WebForms project so want to know if it's going to continue working

**Reply by MichaelKetting** [reply]
We fixed improved Designer dependencies in v7.0.0-alpha.1, so I recommend using that latest pre-prelease. Please also checkout the [3rd party components wiki page](https://github.com/CoreForms/docs/wiki/Compatibility:-3rd-Party-Components) for the libraries we tried / haven't tried yet.

The Bridge is just a bunch of type forwards to make sure we can handle (almost) everything from System.Web that is referenced by 3rd party components so it is meant as a one size fits all. That said, you need to test it with a library but once you confirm it works, there is no reason why it should break. Please let us know once you know it works so we can add it to the wiki page.

**Comment by gsmental** [comment]
what is the roadmap for .net8 and price tag for this product? any license detail?

**Reply by MichaelKetting** [reply]
@gsmental for the pricing, please write to us at CoreForms@rubicon.eu.

For the license, we have a publically available trial license (see https://github.com/CoreForms/docs/wiki/Trial-License). The commerical license depends on the pricing model, but the typical scenario is that you buy a license per developer working on your application's code base and renew this with each major version (i.e. yearly).

The .NET 8 version can be found on nuget.org at https://www.nuget.org/packages/CoreForms.Web/ et al.
